src/libs/antares/study/area/list.cpp

```diff
@@ -1064,13 +1080,27 @@ static bool AreaListLoadFromFolderSingleArea(Study& study,
             auto* section = ini.find("adequacy-patch");
             for (auto* p = section->firstProperty; p; p = p->next)
             {
-                bool value = p->value.to<bool>();
                 CString<30, false> tmp;
                 tmp = p->key;
                 tmp.toLower();
-                if (tmp == "use-adequacy-patch")
+                if (tmp == "adequacy-patch-mode")
                 {
-                    area.bUseAdequacyPatch = value;
+                    auto value = p->value.to<int>();
+                    switch (value)
+                    {
+                    case 0:
+                        area.adequacyPatchMode = Data::adqmVirtualArea;
+                        break;
+                    case 1:
+                        area.adequacyPatchMode = Data::adqmPhysicalAreaOutsideAdqPatch;
+                        break;
+                    case 2:
+                        area.adequacyPatchMode = Data::adqmPhysicalAreaInsideAdqPatch;
+                        break;
+                    default:
+                        area.adequacyPatchMode = Data::adqmVirtualArea;
```

Comment:
Same here, default should be adqmPhysicalAreaOutsideAdqPatch

Reply:
ok